cpp/oneapi/dal/algo/linear_regression/train_types.hpp

```diff
@@ -98,9 +98,47 @@ class train_result {
         return *this;
     }
 
+    /// Table of Linear regression intercept
+    const table& get_intercept() const;
+
+    auto& set_intercept(const table& value) {
```

Comment:
Do we need this?
The same question for set_coefficients, set_packed_coefficients.

Reply:
DIscussed